minicluster/src/main/java/tachyon/underfs/UnderFileSystemCluster.java

```diff
@@ -51,6 +51,16 @@ public void run() {
   private static String sUfsClz;
 
   /**
+   * @return the existing underfs, throwing IllegalStateException if it hasn't been initialized yet
+   */
+  public static synchronized UnderFileSystemCluster get() {
```

Comment:
I see. Make sense. Let's work on adding unit tests of `UnderFileSystemCluster` later.